src/main/scala/org/tessellation/aci/ExplicitKryoRegistrar.scala

```diff
@@ -0,0 +1,26 @@
+package org.tessellation.aci
+
+import cats.Monoid
+import com.twitter.chill.{IKryoRegistrar, Kryo}
+
+case class ExplicitKryoRegistrar(classes: Set[(Class[_], Int)])
+    extends IKryoRegistrar {
+
+  def apply(k: Kryo): Unit =
+    classes.foreach {
+      case (registryClass, id) => k.register(registryClass, id)
+    }
+}
+
+object ExplicitKryoRegistrar {
+  implicit val monoid: Monoid[ExplicitKryoRegistrar] =
+    new Monoid[ExplicitKryoRegistrar] {
+
+      def combine(x: ExplicitKryoRegistrar,
```

Comment:
When we are recompiling the registrar, we want to include all algebras/coalgebras and implicit methods into the Home object